api/v1/deployment/handlers.go

```diff
@@ -96,12 +96,15 @@ func HandlerFindDeployment(appRepo *application.Repository, envRepo *environment
 // HandlerListActiveDeployments list active deployments for a given application version
 func HandlerListActiveDeployments(appRepo application.FindOneByUniqueKey, depRepo *Repository) gin.HandlerFunc {
 	return tonic.Handler(func(c *gin.Context, request *deploymentCreateRequest) (interface{}, error) {
-		app, err := appRepo.FindOneByDomainNameVersion(request.Domain, request.Name, request.Version)
-		if err != nil {
-			return nil, err
+		criteria := map[string]interface{}{}
+		var deps []*v1.Deployment
+		var err error
+		// Search can be done with a version
+		if len(request.Version) > 0 {
```

Comment:
this requires a second handler since the repository logic is no more the same

Reply:
Done, i add two handlers : 
- HandlerListApplicationActiveDeployments
- HandlerListReleaseActiveDeployments